src/lib/image.ts

```diff
@@ -58,8 +56,9 @@ export function readImage(file: File|Blob): Promise<ReadImageReturn> {
       img.src = reader.result as string;
     };
 
-    if (file.type === 'image/heif') {
-      file = await heic2any({ blob: file, toType: 'image/jpeg' }) as Blob;
+    if (file.type === 'image/heif' && file.name.endsWith('.heic') && typeof window !== 'undefined') {
```

Comment:
This condition may seems strange but image/heif MIME type has two variants (.heic and .heif). With .heif for some reason the browser seems to understand the image and does work well without the `heic2any` dependency, but with .heic files crash and that is why there is that dependency. That is why the condition checks explicitly that the file ends with .heic (the File class does not have any option that tells which extension of the MIME type is the file thats why I check the extension from the file name).

And the window check is to run it only on client side.